src/tsc/tsconfig.json

```diff
@@ -1,6 +1,7 @@
 {
     "extends": "../tsconfig-base",
     "compilerOptions": {
+        "types": ["node"]

```

Comment:
not so good, maybe a new reference project

Reply:
This is tsc; all of our executables target node. The compiler itself has these types enabled too.